pkg/model/test_model.py

```diff
@@ -1,4 +1,5 @@
 import os
+from pathlib import Path
```

Comment:
This whole file can be deleted since we don't want to unit test the `model` package.

Reply:
cool, deleting